addons/base_address_city/models/res_partner.py

```diff
@@ -4,11 +4,14 @@
 from lxml import etree
 
 from odoo import api, models, fields
+from odoo.tools.translate import _
+
 
 class Partner(models.Model):
     _inherit = 'res.partner'
 
-    country_enforce_cities = fields.Boolean(related='country_id.enforce_cities')
+    country_enforce_cities = fields.Boolean(
```

Comment:
please avoid this kind of cosmetic changes (by the way, we [do not follow](https://www.odoo.com/documentation/11.0/reference/guidelines.html#python) the E501 of PEP8 guidelines)